docs/create_deploy.md

```diff
@@ -62,7 +61,7 @@ Deploying an instance
 ---------------------
 
 The `Compile tab` displays information related to the current contract
-(note that there can be more than one) (see ../compile\_tab).
+(note that there can be more than one) ([(see compile](compile.html)).
```

Comment:
There is a parenthesis showing where it shouldn't - try this:
(note that there can be more than one) ([see compile](compile.html)).